WordPressKit/ReaderPostServiceRemote.h

```diff
@@ -62,6 +62,45 @@
                success:(void (^)(RemoteReaderPost *post))success
                failure:(void (^)(NSError *error))failure;
 
+/**
```

Comment:
I was thinking if this should really belong to `ReaderPostServiceRemote`.

We're implementing this feature under Reader in the WPiOS app... but thinking about the domain — and not the feature — maybe this can fit better somewhere else? Not sure though. What are your thoughts?

Reply:
Hmmmm. That's a good question.

I decided to define the new subscribe/unsubscribe endpoints in `ReaderPostServiceRemote`, because the like/unlike endpoints are also defined there. I get your point about domain vs feature, and I'm trying to think of a better place, but I'm not so sure either. (Perhaps PostsService..?)

As for you suggestion about adding the new endpoints in Swift, how about I define them in `...ServiceRemote+Subscription.swift`?

Reply:
> Also, wdyt about adding some unit tests to the new endpoints? :)

Will do! 👍

Reply:
> because the like/unlike endpoints are also defined there

Hmm, in that case let's just keep it where is is then! :)

Reply:
Added tests! 8922f20